tasks/npm.yml

```diff
@@ -19,7 +19,7 @@
   - name: NPM dependency install
     # ran into a lot of issues using the ansible npm, shell, command modules
     # had to fall back to raw to press forward
-    raw: cd "{{ django_stack_npm_dir }}" && npm install
+    raw: "cd {{ django_stack_npm_dir }} && {{ django_stack_npm_install_cmd }}"
```

Comment:
dude i would love to stop using raw here... was super annoying problem trying to work around the ansible `npm` module